src/Components/FormGroups/RegisterWithActivationKey.js

```diff
@@ -1,24 +1,24 @@
 import {
   ClipboardCopy,
   FormGroup,
-  Popover,
-  Text,
-  TextContent,
+  // Popover,
+  // Text,
+  // TextContent,
 } from '@patternfly/react-core';
 import React from 'react';
 import CopyHelperText from './CopyHelperText';
-import {
-  OutlinedQuestionCircleIcon,
-  ExternalLinkAltIcon,
-} from '@patternfly/react-icons';
+// import {
+//   OutlinedQuestionCircleIcon,
+//   ExternalLinkAltIcon,
+// } from '@patternfly/react-icons';
 
 const RegisterWithActivationKey = () => {
   return (
     <FormGroup
       label={
         <span>
           Register with an activation key&nbsp;
-          <Popover
+          {/* <Popover
```

Comment:
I am leaving this here for now as I don't know if this was just forgotten in the design. But having Popover over Popover is just bad.